README.md

```diff
@@ -39,33 +39,51 @@ Procedure:
 
 1. Clone the pelux-manifests git repository.
 2. Start vagrant
-```bash
-vagrant up --provider="docker"
-```
+
+    ```bash
+    vagrant up
```

Comment:
For some reason docker seems to be the default provider, both on my computer and on the build server:

    Bringing machine 'default' up with 'docker' provider...